ui/src/main/java/org/pentaho/mantle/client/workspace/SchedulesPanel.java

```diff
@@ -427,10 +427,6 @@ public String getStringValue( JsJob jsJob ) {
             return BLANK_VALUE;
           }
 
-          if ( !GenericFileNameUtils.isRepositoryPath( outputPath ) ) {
```

Comment:
Open the gates!

Reply:
![Untitled](https://github.com/pentaho/pentaho-scheduler-plugin/assets/19820403/5032d267-c697-4070-984b-314d317ed7ff)